tensorflow/compiler/tf2tensorrt/convert/convert_graph.cc

```diff
@@ -957,6 +961,26 @@ tensorflow::Status ConvertAfterShapes(ConversionParams& params) {
   LOG(INFO) << "Number of TensorRT candidate segments: "
             << initial_segments.size();
 
+  // Check if plugins can be aaccessed.
+  int num_trt_plugins = 0;
+  nvinfer1::IPluginCreator* const* trt_plugin_creator_list =
```

Comment:
This ConvertAfterShapes() is too large, can we encapsulate the newly added lines into a helper method and call it here?